.changelog/2436.txt

```diff
@@ -0,0 +1,11 @@
+```release-note:note
+resource/mongodbatlas_cloud_backup_snapshot_export_job: Deprecates the `err_msg` attribute. The attribute is no longer returned from the backend and was considered low value for customers
```

Comment:
i would remove the second sentence, i don't think we normally explain so much in detail in the changelog : "The attribute is no longer returned from the backend and was considered low value for customers"